extensions/arc/src/controller/generated/v1/api.ts

```diff
@@ -0,0 +1,4 @@
+// This is the entrypoint for the package
+export * from './api/apis';
```

Comment:
What are your thoughts on combining the request logic for this and BDC into their own extension? It seems like there's at least some overlap - and not having the duplicated logic would be a good idea I think.

Reply:
I think combining makes sense.  There is overlap for many routes, including at least /token, /info, /endpoints, /control, and /app.  Combining makes sense as we consolidate BDC and Arc.